src/Drivers/nlpMDS_raja_ex4.cpp

```diff
@@ -198,9 +190,9 @@ bool Ex4::get_vars_info(const size_type& n, double *xlow, double* xupp, Nonlinea
       }
     });
 
-  /// Using OpenMP execution policy for nonlinearity type setting
-  RAJA::forall<RAJA::omp_parallel_for_exec>(RAJA::RangeSegment(0, n),
-    [=](RAJA::Index_type i)
+  // Must be a sequential host policy for now
```

Comment:
Make sure people understand this is a temporary solution. 
```suggestion
  // Use a sequential policy for host computations for now
```